### PR TITLE
SAK-46983 LTI rename visible "basiclti" to "lti"

### PR DESCRIPTION
**Pull request by csev**
This renames easily changed visible basic lti values to lti

This does not reorganize the source folders or package names

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/js/show-page.js [hunk 1]
```diff
@@ -2975,6 +2975,7 @@ $(document).ready(function() {
 		enableHighlight: false,
 	});
 
+    // SAK-46983 - Keep for legacy links
```
Comment (ern) [inline lessonbuilder/tool/src/webapp/js/show-page.js:2978]:
```suggestion
```

Comment (ern) [inline lessonbuilder/tool/src/webapp/js/show-page.js:2978]:
```suggestion
    // Content will have the legacy links, so this is needed to process those
```

Comment (ern) [reply lessonbuilder/tool/src/webapp/js/show-page.js:2978]: remove duplication

**Comment by csev** [comment]
Getting no reviews, I am just going to merge this one.

**Pull request merged**